src/deepsparse/yolo/schemas.py

```diff
@@ -41,6 +45,11 @@ class YOLOInput(BaseModel):
         description="List of Images to process"
     )
 
+    @classmethod
+    def from_bytes(cls, bytes: List[bytes]):
+        images = [np.array(Image.open(io.BytesIO(byte_img))) for byte_img in bytes]
```

Comment:
I think that I would pass bytes to the pipeline itself and perform the processing of bytes into numpy array in there. Also, using PIL here (which is not the deepsparse dependency) for quick prototyping, would change it to reduce dependencies.